src/requests/team.ts

```diff
@@ -271,34 +273,55 @@ async function getBySlug(client: GraphQLClient, slug: string) {
   return response.teams[0];
 }
 
-interface GetBopsSubmissionURL {
+interface GetEncryptedIntegrations {
   teams: {
     integrations: {
       bopsSubmissionURL: string | null;
+      bopsSecret: string | null;
     } | null;
   }[];
 }
 
-async function getBopsSubmissionURL(
-  client: GraphQLClient,
-  slug: string,
-  env: PlanXEnv,
-) {
+interface Integrations {
+  bopsSubmissionURL?: string;
+  bopsToken?: string;
+}
+
+/**
+ * Return integration details for a team
+ *
+ * XXX: Why do we need env? Why are there columns for staging and production secrets?
+ * Please see https://github.com/theopensystemslab/planx-new/pull/2499
```

Comment:
It would be good to talk about this at the next dev call - I've added an item to the agenda.